### ISSUE TITLE
Show timer during awards breaks

### ISSUE THREAD

**Issue by jmessing-29** [FIRST-Tech-Challenge/scorekeeper#770]
**Is your feature request related to a problem? Please describe.**
During playoffs, we have to manually track the time between matches.

**Describe the solution you'd like**
We would like a timer to be automatically shown on the audience and field displays as well as on the match control page during the breaks between playoff matches 

**Describe alternatives you've considered**
We have based our time off data available in the cycle time report but would prefer an official time and visual

**Additional context**
N/A

**Comment by emmell620** [comment]
I disagree with showing the time on the audience display. I wouldn't want to detract the award process that shows the finalists and winners with a running clock.  I agree that showing it on the field display and match control page would be helpful, however.

**Comment by jmessing-29** [comment]
I suggested it across all displays as that's what happens in FRC. At our qualifier this weekend, we only used ~half the 15 minute break time for awards. I understand different-sized events have different lengths of breaks. 

Besides that, teams in the pits are looking at our audience displays to know how much time is left. They can't see the field displays at that point and I've been having to chase them down in the pits to start the next match.

**Comment by timtim17** [comment]
Note that, to my understanding, FRC does not show the timer when an Awards display is up. For example, see around the 1:00 mark of this video: https://youtu.be/832p4skoSx4?t=60

When the "Timeout" display is up, of course the timer is visible. But, when an award starts to be presented, it is replaced by the full screen awards display with no visible timer.

Of course, FTC should make the best decision for the program, not just because FRC does it a certain way. Just offering a data point.

**Comment by rachmo** [comment]
@jmessing-29 thank you for filing this enhancement request. I wanted to let you know that timer indicators to improve queuing during playoffs are already on the planned development list.  We don't have an estimated date for implementation (unless a developer wants to chime in on this) but we understand the need for this feature!

**Comment by ToddInIowa** [comment]
+1 
I would like to add to this that in our events we configure our field displays as audience displays so that we can get the game animations. As the displays are configured as audience displays, teams at the field don't get to see the timers. The timer on the audience display for live streams can also be helpful for teams watching the live stream to see what is going on at the field when we can't provide networking between the field and pits. 

Having the timer on an audience display would be very nice to have as a configurable option, for both with and without overlays. For a location with overlays, maybe put the timer at the center top of the display with a reason for the timer. This could also possibly extend into custom timers for other breaks that are needed. 

Thank you! 
Todd Volz

**Comment by erikorlowski** [comment]
A note about the debate of timers on the audience display.

In FRC, at the start of a break, someone on your team needs to walk over to the ref panels and use the time displayed there to start a phone timer and track the break. Otherwise, if you're in your pit making a time critical repair, you have no clue how much time you have.

In those high stress situations, I'd prefer for teams to have all the information possible but including the timer on the alliance display.

**Comment by emmell620** [comment]
After holding my first qualifier yesterday, I would like to strongly urge this request be implemented as soon as possible. To be shown on the field display and/or pit display at least. I will stick with my recommendation to make it configurable to be displayed on the audience display. We were able to do the awards between matches in under 10 minutes and would like to show the audience and teams how much time remains in the 15 minute window.

**Comment by trussell-bpi** [comment]
After running two back to back events this weekend, I would alro like to stongly urge this request to be pushed to the front of the list. Not only will it help teams and the audiance know what is going on, I think it will also help imensly with event flow. I've noticed that once we start talking into the mics for awards, you can see the wind leaving the sails of the teams getting their robots ready for their next match. So that 15 minute timer would also help to remind them that time to the next match is still counting down
